.github/workflows/ci.yml

```
@@ -7,7 +8,7 @@ on:
     branches: [main]
 
 jobs:
-  build:
+  ci: # Ce job est toujours utile pour la qualité du code
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)